### PR TITLE
Move cron script to devops folder to avoid path errors

### PR DESCRIPTION
**Pull request by MSevey**
# PULL REQUEST

## Overview
The current `portals-setup-following` script would throw errors due to the updates to the cronjob handling. This addresses those errors by updating the folder that they are downloaded to during setup. 

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [ ] All new methods or updated methods have clear docstrings
 - [ ] Testing added or updated for new methods
 - [ ] Verify if any changes impact the WebPortal Health Checks
 - [ ] Approriate documentation updated
 - [ ] Changelog file created

### CODE REVIEW

**Review by firyx — APPROVED** [review]
Before we deploy this to all portals, we should test on 1 host that `ansible.builtin.cron` task updates existing cron job with new script path (vs. creating a second cron job).

File: playbooks/tasks/portal-firewall-iptables-block-tor-exit-nodes.yml [hunk 1]
```diff
@@ -36,30 +35,30 @@
...
   ansible.builtin.file:
-    path: "{{ webportal_dir }}/scripts"
+    path: "{{ devops_scripts_dir }}"
     state: directory
 
 - name: Upload script to update tor exit node lists (used by root cron job)
   ansible.builtin.template:
     src: templates/tor-blocklists-update.sh.j2
-    dest: "{{ webportal_dir }}/scripts/tor-blocklists-update.sh"
+    dest: "{{ devops_scripts_dir }}/tor-blocklists-update.sh"
```
Comment (firyx) [inline playbooks/tasks/portal-firewall-iptables-block-tor-exit-nodes.yml:44]: nit: looking at this now, we might want to create another var for script path in `webportals.yml` something like: `block_tor_exit_nodes_script_file: "{{ devops_scripts_dir }}/tor-blocklists-update.sh"`

**Review by firyx — APPROVED** [review]

**Review by ro-tex — APPROVED** [review]

File: playbooks/tasks/portal-firewall-iptables-block-tor-exit-nodes.yml (hunk 1 above)
Comment (MSevey) [reply playbooks/tasks/portal-firewall-iptables-block-tor-exit-nodes.yml:44]: Need to run this test still

run this against any server, no need for sudo/become/lastpass
```
- name: Test cron job idempotency
  ansible.builtin.cron:
    name: "test 1"
    minute: "45"
    job: "echo abc"
```

then on the server crontab -l to check the cronjob is present
then update a la updating the "script path"
```
- name: Test cron job idempotency
  ansible.builtin.cron:
    name: "test 1"
    minute: "45"
    job: "echo def"
```

then crontab -l => we should have just 1 our cron job with echo def (echo abc should be overriden)

then remove the test job either via crontab -e or
```
- name: Test cron job idempotency
  ansible.builtin.cron:
    name: "test 1"
    state: absent
```

Comment (MSevey) [reply playbooks/tasks/portal-firewall-iptables-block-tor-exit-nodes.yml:44]: run and confirmed

**Pull request merged**